### PR TITLE
Introduction and Roman_architecture notebooks

### PR DESCRIPTION
**Pull request by perrygreenfield**

### CODE REVIEW

File: 01-Introduction/01-Introduction.ipynb [hunk 1]
```diff
@@ -0,0 +1,185 @@
+{
```
Comment (nden) [inline 01-Introduction/01-Introduction.ipynb:1]: <p>remants - remnants</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/roman-data-workshop/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='2'/>

Comment (nden) [inline 01-Introduction/01-Introduction.ipynb:1]: <ul><li>Double <code>and</code> in   <code>Specific science and and engineering domains.</code></li><li> <span class="ql-cursor">﻿</span></li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/roman-data-workshop/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='4'/>

File: 13-Roman_Data_Models/13-Roman_Data_Models.ipynb [hunk 2]
```diff
@@ -0,0 +1,624 @@
+{
```
Comment (nden) [inline 13-Roman_Data_Models/13-Roman_Data_Models.ipynb:1]: <p>Add links to repositories as in <a href="https://github.com/spacetelescope/rad" target="_blank">rad</a></p><p> </p><p>Remove <strong>Note on system described below</strong> -&gt; I think the idea is for them to use rad and roman_datamodels </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/roman-data-workshop/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='2'/>

Comment (nden) [inline 13-Roman_Data_Models/13-Roman_Data_Models.ipynb:1]: <p>Which tutorial did you mean in <code>xxx</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/roman-data-workshop/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='15'/>

Comment (nden) [inline 13-Roman_Data_Models/13-Roman_Data_Models.ipynb:1]: <p>Install the latest gwcs in your env to remove the warnings.</p><p> </p><p>And change the name of the <code>cal</code> file </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/roman-data-workshop/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (nden) [inline 13-Roman_Data_Models/13-Roman_Data_Models.ipynb:1]: <p>I wold make the steps a list so it's easier to read and understand</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/roman-data-workshop/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

File: 01-Introduction/01-Introduction.ipynb (hunk 1 above)
Comment (perrygreenfield) [reply 01-Introduction/01-Introduction.ipynb:1]: <p>Done.</p>

Comment (perrygreenfield) [reply 01-Introduction/01-Introduction.ipynb:1]: <p>Done.</p>

File: 13-Roman_Data_Models/13-Roman_Data_Models.ipynb (hunk 2 above)
Comment (perrygreenfield) [reply 13-Roman_Data_Models/13-Roman_Data_Models.ipynb:1]: <p>Removed.</p>

Comment (perrygreenfield) [reply 13-Roman_Data_Models/13-Roman_Data_Models.ipynb:1]: <p>Done, I think...</p>

Comment (perrygreenfield) [reply 13-Roman_Data_Models/13-Roman_Data_Models.ipynb:1]: <p>None now (I wasn't sure if such examples would be in yours).</p>

**Pull request merged**